client/blocks/post-diff-viewer/docs/example.jsx

```diff
@@ -0,0 +1,48 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import PostDiffViewer from 'blocks/post-diff-viewer';
+
+export default function PostDiffViewerExample() {
+	const diff = {
+		title: [
+			{ op: 'copy', value: 'Bitcoin ' },
+			{ op: 'del', value: 'climbs' },
+			{ op: 'add', value: 'rockets' },
+			{ op: 'copy', value: ' above previous high' },
+			{ op: 'del', value: '.' },
+			{ op: 'add', value: '!' },
+		],
+		content: [
+			{ op: 'copy', value: 'Today the price of Bitcoin ' },
+			{ op: 'del', value: 'climbed' },
+			{ op: 'add', value: 'rocketed' },
+			{ op: 'copy', value: ' to over ' },
+			{ op: 'del', value: '$15000' },
+			{ op: 'add', value: '$16000' },
```

Comment:
And to think, it was over 17 just yesterday!

Reply:
Let's connect this example to a realtime API! :D

Reply:
Saving the `<CorrectMyBitcoin />` component for another PR 😄